src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Versioning/CollectionSettingsUpdater.cs

```diff
@@ -59,7 +59,7 @@ public async Task ExecuteAsync(IDocumentClient client, DocumentCollection collec
                         Indexes = new Collection<Index>
                         {
                             new RangeIndex(DataType.Number, -1),
-                            new HashIndex(DataType.String, 3),
+                            new RangeIndex(DataType.String, -1),
```

Comment:
Now the range index on string on the root level, some of the policies below is not longer needed and can be removed.